dev/com.ibm.ws.microprofile.openapi_fat/fat/src/com/ibm/ws/microprofile/openapi/fat/FATSuite.java

```diff
@@ -56,4 +58,8 @@ public static RepeatTests defaultRepeat(String serverName) {
             MicroProfileActions.MP33, // mpOpenAPI-1.1, FULL
             MicroProfileActions.MP22);// mpOpenAPI-1.0, FULL
     }
+
+    static {
+        System.setProperty("sun.net.http.allowRestrictedHeaders", "true");
```

Comment:
Does this need to be set here as well as in the test itself?

Reply:
Also, what's this needed for? Where's it documented? Does it work the same on all Java runtimes?

Unless it's documented officially, we need to run a personal build that runs with a few different Java runtimes.

Reply:
Added an explanation for setting this property, and removed it from the test class as it was redundant

Reply:
Please remove the link.

It's sufficient to say that that the property needs to be set before the `HttpURLConnection` class is first used to allow us to set `Origin` header when making requests.

I could not find any official documentation for this property, but I did find many references to it in forum posts and on Q&A sites, so it appears to be a widely used internal option. I think that's fine for test use, but we should run a build against a variety of Java runtimes to ensure it is supported on all the platforms we test against.